cfgov/ask_cfpb/models/pages.py

```diff
@@ -67,6 +67,24 @@ def get_ask_breadcrumbs(category=None):
     return breadcrumbs
 
 
+def validate_page_number(request, paginator):
+    """
+    A utility for parsing a pagination request,
+    catching invalid page numbers and always returning
+    a valid page number, defaulting to 1.
+    """
+    raw_page = request.GET.get('page', 1)
+    try:
+        page_number = int(raw_page)
+    except ValueError:
+        page_number = 1
+    try:
+        paginator.page(page_number)
```

Comment:
@willbarton I initially tried a similar pattern, passing the paginator around, but it kept breaking the paginator for subsequent requests. 
I could take another swing at it.